server_extensions_api/src/main/java/com/generalbytes/batm/server/extensions/IExtensionContext.java

```diff
@@ -162,6 +168,15 @@
      */
     List<IIdentity> findIdentitiesByPhoneNumber(String phoneNumber, String countryCode);
 
+    /**
+     * Finds and returns identity based on provided phone number. If you don't specify country then number has to be in international format (leading +countrycode is required)
+     *
+     * @param phoneNumber phone number
+     * @param countryCode ISO 3166 Alpha-2 code
+     * @return List of simplified identities (with only basic attributes without collections)
```

Comment:
List of identities (with only basic attributes without related data represented by collections)